knockserver/database.py

```diff
@@ -0,0 +1,25 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import scoped_session, sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+import config
+import os
+
+# Get the database string from config
+db_connect_string = config.DATABASE_URI
```

Comment:
Do we need this extra line?  It looks like it only used once, and you can just substitute it with config.DATABASE_URI one line 10.